ambry-store/src/main/java/com.github.ambry.store/CompactionManager.java

```diff
@@ -154,6 +154,28 @@ void controlCompactionForBlobStore(BlobStore store, boolean enable) {
     }
   }
 
+  /**
+   * Remove store from compaction manager.
+   * @param store the {@link BlobStore} to remove
+   * @return {@code true} if store is removed successfully. {@code false} if not.
+   */
+  boolean removeBlobStore(BlobStore store) {
+    boolean result;
```

Comment:
It's not necessary to keep the result in a local variable unless it is intermediate state.  In all cases here, you can simply return the result.

Reply:
Fixed